### PR TITLE
added one questions in javascript-quiz.ts

### PR DESCRIPTION
**Pull request by jnadroj**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

**Comment by jdwilkin4** [comment]
I removed the `Closes #14 ` from your PR because we want this issue to stay open for multiple contributors. 👍

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1223,6 +1223,17 @@ const javascriptQuiz = [
...
+  { 
+    Question:
+      "In JavaScript, which method allows you to cancel a repeated action created with setInterval()?",
+    Answer: "clearInterval",
+    Distractor1: "clearImmediate",
+    Distractor2: "clearTimeout",
+    Distractor3: "stopInterval",
+    Explanation:
+      "This clearInterval() method is used to stop the timed loop that was started by the setInterval() method.",
+    Link: "https://www.w3schools.com/jsref/met_win_clearinterval.asp",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1235]: We only accept links from freeCodeCamp resources, wikipedia or official documentation. 

```suggestion
    Link: "https://www.freecodecamp.org/news/javascript-timing-events-settimeout-and-setinterval/", 
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jnadroj** [comment]
I've solved the comments!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**